src/evm-config.js

```diff
@@ -7,13 +7,16 @@ const { color } = require('./utils/logging');
 const { ensureDir } = require('./utils/paths');
 const goma = require('./utils/goma');
 
-const configRoot = process.env.EVM_CONFIG || path.resolve(__dirname, '..', 'configs');
-const currentFile = path.resolve(configRoot, 'evm-current.txt');
-let sessionFile;
-if (process.env.EVM_SESSION_ID && !process.env.EVM_SESSION_ID.includes('.')) {
-  sessionFile = path.resolve(configRoot, 'sessions', process.env.EVM_SESSION_ID, 'evm-current.txt');
-}
 const preferredFormat = process.env.EVM_FORMAT || 'json'; // yaml yml json
+const configRoot = process.env.EVM_CONFIG || path.resolve(__dirname, '..', 'configs');
+
+// If you want your shell sessions to each have different active configs,
+// try this in your ~/.profile or ~/.zshrc or ~/.bashrc:
+// export EVM_CURRENT_FILE="$(mktemp --tmpdir evm-current.XXXXXXXX.txt)"
```

Comment:
I would prefer we document this instead of leaving it in a comment - otherwise someone would only be able to find the functionality by looking at this specific file

Reply:
I held off on doing that because it seemed like too much information for a README, but you're right that it needs to be documented.

In 9a46e841785b1188a2b5fc533b0615e8a4e48d59 I've added an 'Advanced Usage' section and made EVM_CURRENT_FILE a subitem underneath that